pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +213,16 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("persistentvolumes", "persistentvolumes/status", "storageclasses").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("events", "limitranges", "namespaces", "namespaces/status",
```

Comment:
@liggitt  +1 ... will fix it up and as I stated, if we find we need to expand later we can...thanks for looking it over